src/promptflow/tests/executor/unittests/executor/test_flow_executor.py

```diff
@@ -403,18 +539,13 @@ class TestGetAvailableMaxWorkerCount:
     @pytest.mark.parametrize(
         "total_memory, available_memory, process_memory, expected_max_worker_count, actual_calculate_worker_count",
         [
-            (1024.0, 128.0, 64.0, 1, -3),    # available_memory - 0.3 * total_memory < 0
+            (1024.0, 128.0, 64.0, 1, -3),  # available_memory - 0.3 * total_memory < 0
             (1024.0, 307.20, 64.0, 1, 0),  # available_memory - 0.3 * total_memory = 0
-            (1024.0, 768.0, 64.0, 7, 7),    # available_memory - 0.3 * total_memory > 0
+            (1024.0, 768.0, 64.0, 7, 7),  # available_memory - 0.3 * total_memory > 0
         ],
     )
     def test_get_available_max_worker_count(
-        self,
-        total_memory,
-        available_memory,
-        process_memory,
-        expected_max_worker_count,
-        actual_calculate_worker_count
+        self, total_memory, available_memory, process_memory, expected_max_worker_count, actual_calculate_worker_count
```

Comment:
Nit: why format changed?

Reply:
Not sure why, it's done by pre-commit directly...